lib/src/date-input/types.ts

```diff
@@ -57,14 +57,14 @@ type Props = {
    * If the string value is a valid date, error will
    * be null. Also, if the string value is not a valid date, date will be null.
    */
-  onChange?: (val: { value: string; error: string | null; date: Date | null }) => void;
+  onChange?: (val: { value: string; error?: string; date?: Date }) => void;
   /**
    * This function will be called when the input element loses the focus.
    * An object including the string value, the error and the date value
    * will be passed to this function. If the string value is a valid date, error will
    * be null. Also, if the string value is not a valid date, date will be null.
    */
```

Comment:
The error portion of the comment needs to be updated as the documentation.

Reply:
Same as the error property below.

Reply:
👍🏼